packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -321,6 +327,15 @@ class UserAccountsDrawerHeader extends StatefulWidget {
   /// upper-right corner. Normally a list of [CircleAvatar] widgets.
   final List<Widget>? otherAccountsPictures;
 
+  /// The size of the widget that placed in the upper-left corner that represents the current
+  /// user's account. Normally a [CircleAvatar].
```

Comment:
The first line should be one simple sentence, so this should be more like this:

```
/// The size of the widget that placed in the upper-left corner that represents the current
/// user's account.
///
/// Normally a [CircleAvatar].
```

Same for the otherAccountsPicturesSize docs below.

Reply:
OK,
I will edit that according to your suggestion.